tests/test_definitions/multicolumn_map_expectations/expect_multicolumn_sum_to_equal.json

```diff
@@ -35,7 +35,7 @@
       }
     },{
       "title": "row_condition_with_ignore_if_any_are_missing",
-      "only_for": ["pandas"],
+      "suppress_test_for": "spark",
```

Comment:
@NathanFarmer Please confirm what I am seeing: your PR enables the metric for `SQL`, hence the expectation is automatically enabled on `SQL`, thereby allowing us to make the statement that only the `Spark` integration test needs to be suppressed.  Thanks.

Reply:
@alexsherstinsky You are correct, and I have verified that logic in `test_expectations_cfe.py` and `test_expectations.py`.